### PR TITLE
Use `ledgerConfiguration` term.

### PR DESCRIPTION
**Pull request by dlongley**

### CODE REVIEW

File: schemas/continuity.js [hunk 1]
```diff
@@ -85,12 +85,51 @@ const blockStatus = {
...
+  additionalProperties: false
+};
+
+const webLedgerEvent = {
+  title: 'WebLedgerEvent',
   type: 'object',
   properties: {
     '@context': schemas.jsonldContext(constants.WEB_LEDGER_CONTEXT_V1_URL),
     type: {
+      // TODO: enum `WebLedgerEvent`
```
Comment (mattcollier) [inline schemas/continuity.js]: Can this be done now?

File: schemas/continuity.js [hunk 2]
```diff
@@ -85,12 +85,51 @@ const blockStatus = {
...
+    type: {
+      // TODO: enum `WebLedgerConfigurationEvent`
+      type: 'string',
+      required: true
+    },
+    ledgerConfiguration: {
+      type: 'object',
+      properties: {
+        type: {
+          // TODO: enum `WebLedgerConfiguration`
```
Comment (mattcollier) [inline schemas/continuity.js]: Can this be done now?

File: schemas/continuity.js [hunk 3]
```diff
@@ -85,12 +85,51 @@ const blockStatus = {
...
 
-const event = {
-  title: 'Continuity Event',
+const webLedgerConfigEvent = {
+  title: 'WebLedgerConfigurationEvent',
+  type: 'object',
+  properties: {
+    '@context': schemas.jsonldContext(constants.WEB_LEDGER_CONTEXT_V1_URL),
+    type: {
+      // TODO: enum `WebLedgerConfigurationEvent`
```
Comment (mattcollier) [inline schemas/continuity.js]: Can this be done now?

**Comment by dlongley** [comment]
The TODOs are related to event validation that we should do in another PR (potentially in the event validation one if we haven't merged it already).

**Pull request merged**